### ISSUE TITLE
tab order (frame)

### ISSUE THREAD

**Issue by b-user** [tnodir/fort#161]
Tab order works, tab index good (yes, my mouse doesn't work). Thanks for the tab order, I can use the tab navigation.
But unfortunately I don’t see the “frame” of the taborder.  Could you please to fix this issue?

**Comment by tnodir** [comment]
@b-user Can you please show a screenshot, where the frame is not shown?

I'm checking a `Program Edit` dialog - current frame is highlighted.

**Comment by tnodir** [comment]
Ah.. these buttons are made specially as link-like buttons and not have tab order.

Maybe it'll be better to set shortcuts for them?

**Comment by tnodir** [comment]
> Okay, a little later I’ll try to describe it in more detail so that it can be reproduced.

Yes, these buttons are not traversable with Tab button.

Ok, let me make them traversable..

**Comment by tnodir** [comment]
Fixed by b952c798.

**Comment by tnodir** [comment]
Please check [v3.10.5-test1](https://github.com/tnodir/fort/releases/tag/v3.10.5-test1).
